posts/copilot-workspace/index.qmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "Some thoughts after a trial run of GitHub’s Copilot Workspace"
+author:
+  - name: "Joshua W. Lambert"
+    orcid: "0000-0001-5218-3046"
+  - name: "James Mba Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Pratik Gupte"
+    orcid: "0000-0001-5294-7819"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-06-24"
+categories: [AI, GitHub, Copilot Workspace, R, R package, software development]
+bibliography: index.bib
+format:
+  html: 
+    toc: true
+---
+
+
+GitHub recently previewed ‘Copilot Workspace’, which aims to use generative AI to assist software developers. Rather than just giving in-line suggestions, as GitHub copilot does, workspace allows users to map out and execute entire projects. We got early preview access to the tool, so decided to see how well it performed for our needs in Epiverse.
+
+In the Epiverse-TRACE initiative we aim to develop an ecosystem of epidemic analytics tools as [Digital Public Goods](https://digitalpublicgoods.net/) in the form of R packages. We aim to [follow best-practices](https://epiverse-trace.github.io/blueprints/) given that our tools will be used to inform  public health decisions. For efficiency, we use a range of development tools, including integrated development environments (IDE), such as RStudio and Visual Studio Code, other R packages to help check and document our code, for example {lintr} and {roxygen2}, and version control and code sharing tools, git and GitHub.
+
+Given the rapid development and increasing availability to artificial intelligence (AI) models, specifically large language models, offer the potential to be another development tool to add to the arsenal of researchers and software engineers to enhance productivity and improve the reliability of the code we write and others depend upon [@vanHoek2024]. Large language models like ChatGPT from OpenAI, Gemini from Google, Claude from Anthropic, as well as many others and new ones frequently appearing, have presented opportunities to quickly generate material -- text, code and images -- from text prompts.
```

Comment:
The first sentence is missing something to make it a sentence but I can't figure it out.